build.xml

```diff
@@ -9,11 +9,11 @@
   <property name="dst.build" location="${dst}/build" />
   <property name="dst.lib" location="${dst}/lib" />
 
-  <property name="groovy.home" value="/usr/share/groovy" />
-  <property name="groovy.version" value="1.8.6" />
-  <property name="java.libs" value="/usr/share/java" />
+  <property name="groovy.home" value="/usr/local/share/groovy" />
+  <property name="groovy.version" value="2.4.12" />
```

Comment:
You have same changes here https://github.com/latera/activiti-ext/pull/2/files. Remove it from one of PRs or you're gonna deal with merge conflicts.